publish.sh

```diff
@@ -19,35 +19,36 @@ RELEASE_NAME=$4
 
 # ensure right number of args
 if [ "$#" -lt 2 ] || [ "$#" -gt 4 ]; then
-    echo $USAGE
+    echo "$USAGE"
     exit 1
 fi
 
 # ensure right kind of arg
 if [ ! -f "${LAMBDA_ZIP}" ]; then
-    echo $USAGE
+    echo "$USAGE"
     echo -e "\nERROR: LAMBDA_ZIP must be a zip file."
     exit 1
 fi
 
-if [ -z ${LAMBDA_NAME} ]; then
-    LAMBDA_NAME=$(basename ${LAMBDA_ZIP})
+if [ -z "${LAMBDA_NAME}" ]; then
+    LAMBDA_NAME=$(basename "${LAMBDA_ZIP}")
     LAMBDA_NAME=${LAMBDA_NAME%.zip}
 fi
 
 BUCKET="${CUMULUS_PREFIX}-artifacts"
 KEY="lambdas/${LAMBDA_NAME}"
-if [ ! -z "${RELEASE_NAME}" ]; then
+if [ -n "${RELEASE_NAME}" ]; then
     KEY="${KEY}-${RELEASE_NAME}"
 fi
 KEY="${KEY}.zip"
 
 echo "Publishing ${LAMBDA_ZIP} to s3://${BUCKET}/${KEY} and lambda ${CUMULUS_PREFIX}-${LAMBDA_NAME}"
 
 # upload to S3
-${AWS_CLI} s3 cp /project/$(basename ${LAMBDA_ZIP}) s3://${BUCKET}/${KEY}
+"${AWS_CLI}" s3 cp "${LAMBDA_ZIP}" s3://"${BUCKET}"/"${KEY}"
```

Comment:
I notice that `/project` was dropped from the path here, was that intentional?

Reply:
hmm no, oops. can I make a commit with that on here directly?

Reply:
Yeah, should be able to via the GitHub GUI or pushing code locally. Here, the suggestion block makes it easy:

```suggestion
"${AWS_CLI}" s3 cp /project/"${LAMBDA_ZIP}" s3://"${BUCKET}"/"${KEY}"
```